OpenSim/Common/Logger.cpp

```diff
@@ -150,6 +149,17 @@ void Logger::addFileSink(const std::string& filepath) {
              "removeFileSink() first.", m_filesink->filename());
         return;
     }
+    // verify if file can be opened at the specified path
+    std::ifstream ifs(filepath);
+
+    if (ifs.is_open()) {
+        ifs.close();
+    } else {
+        // show message:
+         warn("Can't open file {} for writing. Log file will not be created.", 
```

Comment:
Can you make the style of this warning consistent with the one above?

Reply:
Not sure what specifically you want changed, I took a stab at it. As a one liner, feel free to change as well as you see fit. Thank you.